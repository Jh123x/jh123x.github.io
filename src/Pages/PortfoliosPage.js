import React from "react";
import { MainLayout, InnerLayout } from "../styles/Layouts";
import Title from "../Components/Title";
import portfolios from "../data/portfolios";
import Menu from "../Components/Menu";

const PortfoliosPage = () => (
  <MainLayout>
    <Title title={"Portfolios"} span={"portfolios"} />
    <InnerLayout>
      <Menu menuItem={portfolios} />
    </InnerLayout>
  </MainLayout>
);

export default PortfoliosPage;
