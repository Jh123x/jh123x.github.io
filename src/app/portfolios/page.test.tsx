import * as React from "react";
import PortfoliosPage from "./page";
import { describe, expect, test } from "@jest/globals";

describe("Portfolio Page", () => {
  test("should render correctly", () => {
    const page = <PortfoliosPage />;
    expect(page).toMatchSnapshot();
  });
});
