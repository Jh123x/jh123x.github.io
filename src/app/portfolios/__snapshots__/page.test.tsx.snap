// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio Page should render correctly 1`] = `<PortfoliosPage />`;
